<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controls</class>
 <widget class="QWidget" name="Controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>999</width>
    <height>79</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSlider" name="horizontalSliderVolume">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSliderPosition">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLCDNumber" name="lcdNumberElapsed">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
color:red;
}</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>8</number>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumberDuration">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonPrev">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/resources/prev.svg</normaloff>:/resources/prev.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPlayPause">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/resources/pause.svg</normaloff>:/resources/pause.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonNext">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/resources/next.svg</normaloff>:/resources/next.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonPrev</sender>
   <signal>clicked()</signal>
   <receiver>Controls</receiver>
   <slot>onPrevClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>736</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPlayPause</sender>
   <signal>clicked()</signal>
   <receiver>Controls</receiver>
   <slot>onPlayPauseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>762</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>464</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonNext</sender>
   <signal>clicked()</signal>
   <receiver>Controls</receiver>
   <slot>onNextClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>821</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderVolume</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Controls</receiver>
   <slot>onVolumeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onVolumeChanged(int)</slot>
  <slot>onPlayPauseClicked()</slot>
  <slot>onStopClicked()</slot>
  <slot>onNextClicked()</slot>
  <slot>onPrevClicked()</slot>
  <slot>onElapsedChanged(qint64)</slot>
  <slot>onElapsedChanged(int)</slot>
 </slots>
</ui>
